src/core/Akka.Cluster/ClusterHeartbeat.cs

```diff
@@ -258,8 +258,8 @@ private void DoHeartbeatRsp(HeartbeatRsp rsp)
         {
             if (_cluster.Settings.VerboseHeartbeatLogging)
             {
-                // TODO: log response time and validate sequence nrs once serialisation of sendTime is released
-                _log.Debug("Cluster Node [{0}] - Heartbeat response from [{1}]", _cluster.SelfAddress, rsp.From.Address);
+                _log.Debug("Cluster Node [{0}] - Heartbeat response from [{1}] - Sequence number [{2}] - Creation time [{3}]", _cluster.SelfAddress, rsp.From.Address,
+                    rsp.SequenceNr.ToString(CultureInfo.InvariantCulture), rsp.CreationTimeNanos.ToString(CultureInfo.InvariantCulture));
```

Comment:
Why .ToString()
Because if we pass simple rsp.SequenceNr it will be boxed as its type is long.
Why .ToString(CultureInfo.InvariantCulture)
because ToString is culture dependent

Reply:
Sorry, can't get you.
`MonotonicClock` is just a `Stopwatch`, so it's something like `TimeSpan` since cluster start.
If i get ticks, how it can be converted to `DateTime`?
if i get ticks 5 min after cluster start it will be something like  `January 1, 0001 00:00:05`